README.md

```diff
@@ -1,7 +1,13 @@
-# Terra External Credentials Manager
+# Terra External Credentials Manager (ECM)
 ### An external data credential management service.
-###### (née Kong -> King Kong -> gorilla -> gorilla glue -> a strong bond)
 
-[Design doc in-progress here](https://docs.google.com/document/d/1vNYCAcyxB5bDwR0KIfKEb4h6kIz52BUGyJTIquzq5bg/edit)
+## Helpful Links
 
-###### Dependency locking is turned on, to update the lockfiles run `./gradlew dependencies --write-locks`
+* [Design Doc](https://docs.google.com/document/d/1vNYCAcyxB5bDwR0KIfKEb4h6kIz52BUGyJTIquzq5bg/edit)
+* [Developer Setup Guide](/DEVELOPMENT.md)
+* [Integration & Performance Testing Guide](/integration/README.md)
+* [Swagger Page (Dev)](https://externalcreds.dsde-dev.broadinstitute.org/)
+
+## Other Notes
+
+Dependency locking is turned on, to update the lockfiles run `./gradlew dependencies --write-locks`
```

Comment:
Nit: Could you also add this note in `integration/README.md`?

Reply:
Yeah definitely, thanks for the suggestion!